docs/user-guide/security-features.md

```diff
@@ -1,20 +1,112 @@
+
+![Figure 4: Pre-CI Code Scan Results](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/code-scan.gif)
+
+### After Building Container Image
+
+Once a container image is ready, you can scan its base image libraries, stale files, compromised licenses, and many more.
+
+There are 2 options available:
+* Image scan in the Build stage (refer [Security Scans](./security-features/security-scans.md))
+* Comprehensive scan in Post-Build stage
 
-Devtron's security feature consists of two primary components:
+This section contains the steps for comprehensive scan.
 
-1. [Security Scans](./security-features/security-scans.md) - This allows you to identify and address potential security risks effectively. By leveraging this feature, you can ensure that your containerized applications are safeguarded against known vulnerabilities.
+![Figure 5: Scanning in Post-CI Stage](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/post-ci.gif)
 
-2. [Security Policies](./security-features/security-policies.md) - This allows you to define policies to block or allow the deployment of container images depending on the vulnerabilities detected.
+1. Go to the **Post-build stage** (tab) of your CI pipeline.
+
+2. Click **+ Add Task** and choose **Vulnerability_Scanner v1.0.0**.
+
+3. Click **Update Pipeline**.
+
+Results of Post-CI scan will be visible under `Image Scan` in the **App Details** page as shown below.
+
+![Figure 6: Post-CI Image Scan Results](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/image-scan-1.gif)
+
+### Before Triggering Deployment
+
+There can be a loophole where the original image built in the CI stage gets compromised later (say, in publicly accessible repository). Therefore, you can scan the image and catch issues before deploying it. On top of that, you can also scan manifests to detect misconfigurations and exposed secrets.
+
+![Figure 7: Scanning in Pre-CD Stage](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/pre-deploy.gif)
+
+1. Go to the **Pre-Deployment stage** (tab) of your CD pipeline.
+
+2. Click **+ Add Task** and choose **Vulnerability_Scanner v1.0.0**.
+
+3. Click **Update Pipeline**.
+
+Results of Pre-CD scan will be visible under `Image Scan` and `Kubernetes Manifest` in the **App Details** page as shown below.
+
+![Figure 8: Pre-CD Scan Results](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/manifest-scan.gif)
+
+### During Helm App Deployment
+
+When you [deploy a helm chart](../user-guide/deploy-chart/deployment-of-charts.md), Devtron will scan the image associated with that helm chart and also the manifests, but unlike Devtron Apps, there is no code scan involved.
+
+Results of helm app scan will be visible under `Image Scan` and `Kubernetes Manifest` in the **App Details** page as shown below.
```

Comment:
GIF and steps needed for how to scan, a helm app

Reply:
As per latest discussion, the **Configure** tab of Helm Apps is yet to have scanning options. Currently, it's scanned automatically; therefore, steps for end-user is not available.